contest/contest_info.py

```diff
@@ -110,6 +115,8 @@ def is_contestant(user,contest):
 
 def can_ask(user,contest):
```

Comment:
BTW, is it needed to check user is authenticated before calling can_ask in filter?